src/permalink/Settings.js

```diff
@@ -117,7 +140,7 @@ class Settings {
     }
 
     async loadSettingsFromRepo() {
-        const response = await fetch('https://raw.githubusercontent.com/ssrando/ssrando/1843d46a5561bb7a0d9d1826b23e91bdd984c52d/options.yaml');
+        const response = await fetch('https://raw.githubusercontent.com/ssrando/ssrando/master/options.yaml');
         const text = await response.text();
         this.allOptions = yaml.load(text);
```

Comment:
Update `allOptions` here to set the choices for excluded locations so it doesn't need special handling. This code will remain even past the logic updates so it needs to be solid.